### ISSUE TITLE
GlobalValueEnumeration crash in GADMaxAdContentRating : related to MobiVM/robovm issue #567?

### ISSUE THREAD

**Issue by TheScottyJr** [dkimitsa/robovm-robopods#15]
Updating to latest https://github.com/firebase/firebase-ios-sdk/releases/tag/v8.12.1 and have the GADRewardedAds working, but crashing when setting GADMAxAdContentRating. Not crashing when only tagging for child directed treatment. Possibly related to core robovm issue #567 regarding GlobalValueEnumeration.

https://github.com/dkimitsa/robovm-robopods/blob/41a086b4ef911074e442feefdce7d300f9442d9a/firebase/ios-google-mobile-ads/src/main/java/org/robovm/pods/google/mobileads/GADMaxAdContentRating.java#L106-L114

`java.lang.Error: java.lang.reflect.InvocationTargetException
	at org.robovm.rt.bro.LazyGlobalValue.value(LazyGlobalValue.java:95)
	at org.robovm.apple.foundation.GlobalValueEnumeration.value(GlobalValueEnumeration.java:29)
	at org.robovm.pods.google.mobileads.GADMaxAdContentRating$Marshaler.toNative(GADMaxAdContentRating.java:62)
	at org.robovm.pods.google.mobileads.GADRequestConfiguration.$m$setMaxAdContentRating$(Native Method)
	at org.robovm.pods.google.mobileads.GADRequestConfiguration.setMaxAdContentRating(GADRequestConfiguration.java)
	at com.sjrcode.IOSManager.setAge(IOSManager.java:1320)
	at com.sjrcode.LevelScreen.touchDown(LevelScreen.java:3650)
	at com.badlogic.gdx.backends.iosrobovm.IOSInput.processEvents(IOSInput.java:635)
	at com.badlogic.gdx.backends.iosrobovm.IOSGraphics.draw(IOSGraphics.java:352)
	at com.badlogic.gdx.backends.iosrobovm.IOSGraphics$1.draw(IOSGraphics.java:229)
	at com.badlogic.gdx.backends.iosrobovm.IOSGraphics$1.$cb$drawRect$(IOSGraphics.java)
	at org.robovm.apple.uikit.UIApplication.main(Native Method)
	at org.robovm.apple.uikit.UIApplication.main(UIApplication.java:446)
	at com.sjrcode.IOSLauncher.main(IOSLauncher.java:61)
Caused by: java.lang.reflect.InvocationTargetException
	at java.lang.reflect.Method.internalInvoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:591)
	at org.robovm.rt.bro.LazyGlobalValue.value(LazyGlobalValue.java:89)
	... 13 more
Caused by: java.lang.UnsatisfiedLinkError: Optional @GlobalValue method org/robovm/pods/google/mobileads/GADMaxAdContentRating$Values.MatureAudience()Lorg/robovm/apple/foundation/NSString; not bound
	at org.robovm.pods.google.mobileads.GADMaxAdContentRating$Values.MatureAudience(Native Method)`

**Comment by dkimitsa** [comment]
from the crash log it seems like you using robovm v2.3.12 (which is 1+ y old). 
anyway could you please share code snippet that causes the issue ? 
thx !

**Comment by dkimitsa** [comment]
this is regression caused by removed exported symbols from `robovm.xml`. 
have re-deployed 1.21.0-SNAPSHOT with the fix. 
meanwhile if you use v1.20.0-SNAPSHOT you can call following to your app's robovm.xml
```
    <exportedSymbols>
        <!-- workaround: these will keep symbols from being stripped -->
        <symbol>GAD*</symbol>
    </exportedSymbols>
```

**Comment by TheScottyJr** [comment]
I can confirm this resolved my issue on roboVM v2.3.15(updated from 2.3.12 when you mentioned it) with firebase v8.13.0 libs and altpods v1.21.0-SNAPSHOT.

Adding the workaround myself works as well on robo_2.3.15, fire_8.12.1, and altp_1.20.0-snapshot

Thanks!
